packages/frontend/src/features/comments/hooks/useDashboardCommentsCheck.ts

```diff
@@ -5,16 +5,16 @@ import { UserWithAbility } from '../../../hooks/user/useUser';
 export const useDashboardCommentsCheck = (
     user: UserWithAbility | undefined,
 ) => {
-    const isDashboardTileCommentsFeatureEnabled = useFeatureFlagEnabled(
+    const isDashboardTileCommentsFeatureEnabled = !!useFeatureFlagEnabled(
```

Comment:
This should be updated to use Lightdash's own `useFeatureFlagEnabled` hook (under `src/hooks`), which is correctly typed with the `FeatureFlags` enum and always returns a `boolean` (and not `boolean | undefined`).

Reply:
done!